modules/nw-configure-sysctl-interface-sriov-network-bonded.adoc

```diff
@@ -0,0 +1,204 @@
+         "net.ipv4.conf.IFNAME.disable_policy": "1",
+         "net.ipv4.conf.IFNAME.secure_redirects": "0",
+         "net.ipv4.conf.IFNAME.send_redirects": "0",
+         "net.ipv6.conf.IFNAME.accept_redirects": "0",
+         "net.ipv6.conf.IFNAME.accept_source_route": "1",
+         "net.ipv6.neigh.IFNAME.base_reachable_time_ms": "20000",
+         "net.ipv6.neigh.IFNAME.retrans_time_ms": "2000"
+       }
+       }
+      }'
+----
+<1> The type is `bond`.
+<2> The `ifname` attribute specifies the name of the bond interface.
+<3> The `mode` attribute specifies the bonding mode. The bonding modes supported are:
+
+ * `balance-rr` - 0 
+ * `active-backup` - 1
+ * `balance-xor` - 2
++
+For `balance-rr` or `balance-xor` modes, you must set the `trust` mode to `on` for the SR-IOV virtual function.
+<4> The `failover` attribute is mandatory for active-backup mode.
+<5> The `linksInContainer=true` flag informs the Bond CNI that the interfaces required are to be found inside the container. By default Bond CNI looks for these interfaces on the host which does not work for integration with SRIOV and Multus.
+<6> The `links` section defines which interfaces will be used to create the bond. By default, Multus names the attached interfaces as: "net", plus a consecutive number, starting with one. 
+<7> A configuration object for the IPAM CNI plug-in as a YAML block scalar. The plug-in manages IP address assignment for the attachment definition.
+<8> Optional: The metaPlugins parameter is used to add additional capabilities to the device. In this use case set the `type` field to `tuning`. Specify the interface-level network sysctl you want to set in the sysctl field. This examples sets all interface-level network sysctls. 
+
+. Create the `SriovNetwork` resource:
++
+[source,terminal]
+----
+$ oc create -f sriov-network-interface-sysctl-all.yaml
+----
+
+.Verifying that the `NetworkAttachmentDefinition` CR is successfully created
+
+* Confirm that the SR-IOV Network Operator created the `NetworkAttachmentDefinition` CR by running the following command.
++
+[source,terminal]
+----
+$ oc get network-attachment-definitions -n <namespace> <1>
+----
+<1> Replace `<namespace>` with the namespace that you specified when configuring the network attachment, for example, `openshift-sriov-network-operator`.
++
+.Example output
+[source,terminal]
+----
+NAME                                  AGE
+openshift-sriov-network-operator      14m
+----
++
+[NOTE]
+====
+There might be a delay before the SR-IOV Network Operator creates the CR.
+====
+
+.Verifying that the additional SR-IOV network attachment is successful
+
+To verify that the tuning CNI is correctly configured and the additional SR-IOV network attachment is attached, do the following:
+
+. Create a pod such as `examplepod.yaml` similar to the following:.
++
+[source,terminal]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: tunepod
+  namespace: default
+  annotations:
+    k8s.v1.cni.cncf.io/networks: |-
```

Comment:
For the network-attachment-defintion above we need 3 networks here:
- 2 that define the slaves, we can use sriov networks for these
- 1 for the bond network defined in the network-attachment-defintion above

Reply:
I updated with YAML from @kononovn